### ISSUE TITLE
请问作者，loss中对x,y使用的交叉熵是否应为squared loss？

### ISSUE THREAD

**Issue by ld909** [SpikeKing/keras-yolo3-detection#21]
yolo v3 于**2.1. Bounding Box Prediction**中对xy,wh使用的loss进行了说明：

> During training we use sum of squared error loss.

但在作者的实现中，对wh使用的确实是squared loss，但xy实现如下：
```
xy_loss = object_mask * box_loss_scale * K.binary_crossentropy(raw_true_xy, raw_pred[..., 0:2],
                                                               from_logits=True)
```

所以请教作者为何这样实现？
谢谢
